src/plotter_disk.hpp

```diff
@@ -146,35 +166,41 @@ class DiskPlotter {
             assert(id_len == kIdLen);
 
             // Memory to be used for sorting and buffers
-            uint8_t* memory = new uint8_t[memorySize];
```

Comment:
@mariano54 is there a good reason to clear the memory here? this adds a noticeable delay on startup

Reply:
is the idea to pre-fault the memory into physical RAM and the TLB? I think that would only help with latency later, not necessarily increase throughput